src/http/post-preview-000u-000r/_verify.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const data = require('@begin/data');
+let { get } = require('tiny-json-http');
+
+/**
+ * Verify that the received payload is indeed a real PR + SHA
+ * If so, pass it along
+ */
+module.exports = async function verify (req) {
+  try {
+    const { u: user, r: repo } = req.pathParameters;
+    const { pr, sha } = req.body;
+
+    const base = 'https://api.github.com';
+    const url = `${base}/repos/${user}/${repo}/pulls/${pr}/commits`;
+
+    const commits = await get({ url });
+
+    // Is this an actual SHA on the repo?
+    const shaIsReal = commits.body.find(c => c.sha === sha);
+
+    // Have we seen this SHA before and published its preview?
+    const shaHasBeenSeen = await data.get({
```

Comment:
is this about the same as using `count`?

Reply:
Count is a query, which assumes multiple results would be feasible – the key on this is the sha, so it should be individual.

Reply:
mainly just asking if "count is 0" was more efficient than "get the entire data and ignore it"